docs/custom-transforms-and-renditions.md

```diff
@@ -1,29 +1,30 @@
 # Custom Transforms and Renditions
 
-Alfresco Content Services (ACS) provides a number of content transforms,
-but also allows custom transforms to be added. This section describes
-how to create custom transforms.
+## Transformation changes from Content Services 6.2
 
-From ACS 6.2 it is possible to create custom transforms that run in
+Alfresco Content Services provides a number of content transforms.
+
+From Content Services 6.2 it is possible to create custom transforms that run in
 separate processes known as T-Engines (short for Transformer Engines).
 The same engines may be used
 in Community and Enterprise Editions. They may be directly
-connected to the ACS repository as Local Transforms, but in the
+connected to the Repository as Local Transforms, but in the
 Enterprise edition there is the option to include them as part of the
-Alfresco Transform Service (ATS) which provides more balanced throughput.
+Alfresco Transform Service (ATS), which provides a more balanced throughput.
+
 A T-Engine is intended to be run as a 
 Docker image, but may also be run as a standalone process.
 
-Prior to ACS 6.0 Legacy transformers ran within the same JVM as
-the ACS repository. They and their supporting classes were deprecated
-and in ACS 6, but have now been removed. ACS 6.2 still uses them if a rendition
+Prior to Content Services 6.0 Legacy transformers ran within the same JVM as
+the Repository. They and their supporting classes were deprecated
+in Content Services 6.0 and have now been removed. Content Services 6.2 still uses them if a rendition
 cannot be created by the Transform Service or Local Transforms. The
```

Comment:
docs-alfresco used "Repository" rather than "ACS repository" and "Content Services" rather than "ACS".